### ISSUE TITLE
Removing schema reference in response

### ISSUE THREAD

**Issue by aleksicmarija** [danielgtaylor/huma#428]
Is it possible to configure removing this from response?
`"$schema":"http://localhost:8000/api/v1/schemas/SomeSchema.json"`

**Comment by danielgtaylor** [comment]
@aleksicmarija yes, you can stop using the `SchemaLinkTransformer` that is present in the default config. See here:

https://github.com/danielgtaylor/huma/blob/main/defaults.go#L53

You can either define your own config yourself, or use the default but reset the create hooks to prevent the link transformer from running.

**Comment by aleksicmarija** [comment]
Thanks!

**Comment by leonklingele** [comment]
I guess this can be closed now.

**Comment by IcyFoxe** [comment]
Would be nice to have some mention of this in the documentation.

**Comment by wolveix** [comment]
Docs added, thank you!
